LBH.AdultSocialCare.Api/V1/UseCase/Payments/Concrete/GetPackagePaymentHistoryUseCase.cs

```diff
@@ -36,8 +36,7 @@ public async Task<PackagePaymentViewResponse> GetAsync(Guid packageId, RequestPa
             var payRunStatuses = new[] { PayrunStatus.Approved, PayrunStatus.Paid, PayrunStatus.PaidWithHold };
             var payRunTypes = new[]
             {
-                PayrunType.ResidentialRecurring, PayrunType.DirectPayments, PayrunType.ResidentialReleasedHolds,
-                PayrunType.DirectPaymentsReleasedHolds
+                PayrunType.ResidentialRecurring, PayrunType.ResidentialReleasedHolds
```

Comment:
Why do we need this? GetPackageLatestPayRunAsync should return all payruns. It might be good to remove parameters from function.

Reply:
This is for a different page 
<img width="379" alt="image" src="https://user-images.githubusercontent.com/81692554/146150480-d127d2d4-f621-435f-9811-40d211fd9fd5.png">